tests/foreman/cli/test_remoteexecution.py

```diff
@@ -474,6 +474,12 @@ def test_positive_run_receptor_installer(self, default_sat):
 
         :BZ: 1818076
         """
+        # Copy foreman-proxy user's key to root@localhost user's authorized_keys
```

Comment:
There is a function for this in  the unstable branch, that can be used here or within  a fixture that multiple tests could re-use:

https://github.com/SatelliteQE/robottelo/blob/unstable/robottelo/hosts.py#L491

Reply:
@lhellebr The `unstable` branch has been merged to master, please use the `ContentHost.add_rex_key`

For your implementation here, it would look like `default_sat.add_rex_key(satellite=default_sat)`